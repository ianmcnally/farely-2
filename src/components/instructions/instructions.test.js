import React from 'react'
import Instructions from './instructions'
import renderShallow from 'render-shallow'

describe('with required props', () => {
  let component

  beforeAll(() => {
    component = renderShallow(<Instructions />).output
  })

  it('renders the correct instructions', () => {
    expect(component).toMatchSnapshot()
  })

})

describe('with props.hide=true', () => {
  const props = { hide: true }
  let component

  beforeAll(() => {
    component = renderShallow(<Instructions {...props} />).output
  })

  it('renders with a hidden class', () => {
    expect(component).toMatchSnapshot()
  })
})

