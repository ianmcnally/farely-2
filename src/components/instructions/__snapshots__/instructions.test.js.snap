// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with props.hide=true renders with a hidden class 1`] = `
<p
  className="f5 lh-copy measure-narrow dn"
>
  Enter your MetroCard’s current balance to calculate the amount to put on it for an exact number of rides.
</p>
`;

exports[`with required props renders the correct instructions 1`] = `
<p
  className="f5 lh-copy measure-narrow"
>
  Enter your MetroCard’s current balance to calculate the amount to put on it for an exact number of rides.
</p>
`;
